ohara-manager/client/src/components/common/Definitions/ArrayDef.js

```diff
@@ -32,26 +32,30 @@ const StyledAutocomplete = styled(Autocomplete)(
 
 const ArrayDef = props => {
   const {
-    input: { name, onChange, value: formValue = [] },
+    input: { name, onChange, value: options },
     meta = {},
     helperText,
     refs,
     ...rest
   } = omit(props, ['tableKeys']);
 
-  const hasError = (meta.error && meta.touched) || (meta.error && meta.dirty);
+  if (!options) return <></>;
```

Comment:
Shouldn't we just return a `null`
```suggestion
  if (!options) return null;
```